src/language/server.ts

```diff
@@ -275,9 +277,17 @@ connection.onCompletionResolve(
 /**
  * Execute the entire playground script.
  */
-connection.onRequest('executeAll', (event) => {
-  // connection.console.log(`executeAll: ${JSON.stringify(event)}`);
-  return '';
+connection.onRequest('executeAll', async (params) => {
+  try {
+    const connectionOptions = params.connectionOptions || {};
+    const runtime = new ElectronRuntime(
+      await CliServiceProvider.connect(params.connectionString, connectionOptions)
+    );
+
+    return await runtime.evaluate(params.codeToEvaluate);
```

Comment:
I think we could move this error into a new try that way a code error can be better handled. Currently if there's an error in a playground's code, like calling a method that doesn't exist the error message is `Unable to connect to MongoDB instance. Make sure it is started correctly.`. Does the runtime throw an error that we can use?

Reply:
Removed try-catch at all, since they are being caught in the playground controller anyway.